setup.py

```diff
@@ -36,6 +36,9 @@
     ],
     description='Pythonic IGES reader',
     url='https://github.com/pyvista/pyiges',
-    install_requires=['tqdm', 'geomdl', 'pyvista>=0.28.0'],
+    install_requires=['tqdm'],
+    extras_require={
+        'full' : ['geomdl', 'pyvista>=0.28.0']
+    },
```

Comment:
Here the dependecies are defined flexibly. I locally tried to upload via `test.pypi.org` and it behaves as expected.